core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -235,7 +261,7 @@ object ZValidation extends LowPriorityValidationImplicits {
   /**
    * Derives an `Equal[ZValidation[W, E, A]]` given an `Equal[A]`.
    */
-  implicit def ZValidationEqual[W, E: Equal, A: Equal]: Equal[ZValidation[W, E, A]] =
+  implicit def ZValidationEqual[W, E, A: Equal]: Equal[ZValidation[W, E, A]] =
     Equal.make {
       case (Failure(_, e), Failure(_, e1)) => e == e1
```

Comment:
`===` here, right?

Reply:
Welp, we're comparing two `NonEmptyMultiSet`s, which are just a `Map[X, Int]` inside. From that point of view, `===` is more elegant, but unnecessary ceremony possibly even with some performance overhead.

But if you still think, `===` would be better, I'm not against, I can put it there.

Reply:
Ah, sorry, I thought you were adding the `Equal` construct there. Yes, it doesn't really make sense since we don't have a way to compare equality on the keys with a custom notion of equality efficiency so no reason to pretend that we do.

Reply:
Exactly. Sometimes I wish we just had "our own" ZHashMap and ZOrdMap which would be with proper variance and used our Type classes, like `Hash` and `Ord`...

Reply:
Yeah. That would be a significant project. I'm not sure it is worth it since those are complex data structures where it would be easy to introduce bugs or performance regressions and I don't think there is really a benefit outside of this but I have thought the same thing.